test/make.js

```diff
@@ -0,0 +1,25 @@
+const assert = require('assert');
+const make = require('../task/make.js');
+
+describe('make', () => {
+  it('0', () => {
+    const count = make(15)(34, 21, 666)(41)((a, b) => a + b);
+    assert.deepEqual(count, 777);
+  });
+  it('1', () => {
```

Comment:
The point of having several test for the same code (same function) is to test **different scenarios**
The tests 1 - 4 in your case basically test the same scenario. Which means that 3 of them are actually redundant.
To make these tests more meaningful, you could test some diffeferent scenarios, let's say:
- make(15, 34, 21, 666, 41, (a, b) => a + b);  // all parameters in a one call
- make((a, b) => a + b);  // only the function
- make();  // the **negative** test. The function should work without errors.
- make(15)(1)((a, b) => **a * b**);  // works as expected with some different function.

Basically, the more **different** tests you have, the more chances you will detect the incorrect behavior in your code. And therefore, less chances that it would be found by somebody else 😎